src/js/components/RadioButton/RadioButton.js

```diff
@@ -102,7 +102,7 @@ const RadioButton = forwardRef(
                     viewBox="0 0 24 24"
                     preserveAspectRatio="xMidYMid meet"
                   >
-                    <circle cx={12} cy={12} r={6} />
+                    <circle cx={12} cy={12} r={theme.radioButton.check.size} />
```

Comment:
I know this is being picky, but I wonder if `radius` would be more appropriate here? We aren't using any of our string T-shirt sizes and I think pretty much everywhere else, `size` would be used for the width and height.

Reply:
I agree with @ericsoderberghp, we're talking about a circle element so it makes sense to be `radius` if @ShimiSun or @jcfilben don't have a different thought I will change it.

Reply:
Hey, I found a problem with this rename, we already have a `radioButton.check.radius` that it's for `border-radius` of `check`, So how do you guys want to proceed with this?  
https://github.com/grommet/grommet/blob/85e3d85896327d9e6bf96ca5e6e307bd4793fccc/src/js/components/RadioButton/doc.js#L89

Reply:
Good catch. I agree it's a little tricky, after putting some thoughts into this issue, I wish we had defined the current api with `radioButton.check.border.radius` instead of `radioButton.check.radius`. but from backward compatibility reasons, it will be tricky to override it now while keeping a backward compatibility behavior.

I think the best way to enhance the api while keeping a clean implementation and backward compatible behavior is to use `radioButton.check.size.radius`.

@jcfilben & @ericsoderberghp any thoughts on the new api suggestions?

Reply:
I agree with Shimi that the best thing to do here is to use `radioButton.check.size.radius`